### PR TITLE
Make Client properly encode parameters in query string

### LINKED ISSUE #21: Make to sure to encode query parameters in HTTP call
eg with the workflow id:

louis.cibot+1@gmail.com the kill does not work.

### PR DESCRIPTION
**Pull request by pylebecq**
### Summary

This PR ensures parameters sent in query string are properly encoded by the `Client` class.
This fixes the inability to kill workflows having special characters in their id.

### Safety check

| Q                         | A
| ------------------------- | ---
| Type                      | bugfix
| Related issues            | closes #21 
| Code covered with tests?  | yes
| Changelog updated?        | yes
| Documentation updated?    | no

### CODE REVIEW

**Review by Ciboulette — APPROVED** [review]
Seems good

**Review by idabmat — COMMENTED** [review]
Thanks for bringing this to my attention. I will implement this in the Ruby library too

File: src/Zenaton/v1/Client.php [hunk 1]
```diff
@@ -306,4 +359,17 @@ protected function addAppEnv($url, $params = '')
             .($this->appId ? self::APP_ID.'='.$this->appId.'&' : '')
             .($params ? $params.'&' : '');
     }
+
+    protected function addQueryParams($url, array $params = [])
+    {
+        // When called from worker, APP_ENV and APP_ID is not defined
```
Comment (pylebecq) [inline src/Zenaton/v1/Client.php]:
```suggestion
        // When called from worker, APP_ENV and APP_ID are not defined
```

**Comment by pylebecq** [comment]
Maybe I should add a deprecation warning in the library and make sure I make some changes in the agent to use the proper escaping of parameters. See zenaton/zenaton-ruby#51 and zenaton/agent#175

**Pull request merged**